docs/semgrep-app/scm.md

```diff
@@ -10,14 +10,24 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Integrating Semgrep into source code management (SCM) tools

```

Comment:
What are actually SCM tools? GitHub and GitLab are websites that use one specific tool - Git. But what about Mercurial... not sure here. Maybe we should define this in the introduction at least? Do we mean hosted solutions on the web which use Git and we support them?

![Screenshot 2022-07-01 at 11 41 37](https://user-images.githubusercontent.com/26669307/176869893-895ebc42-9a55-4939-95c5-a8378f830df8.png)

Reply:
@adamkvitek See the top of the article: 

> Integrating Semgrep into source code management (SCM) tools
> Semgrep App's Team tier supports repositories hosted on both GitHub Enterprise (GHE) and GitLab Self Managed (GLSM) plans.

Reply:
@s-santillan Yes, I have seen it. It is still not a plain definition/explanation though. Maybe:

Original:
```
Integrating Semgrep into source code management (SCM) tools
Semgrep App's Team tier supports repositories hosted on both GitHub Enterprise (GHE) and GitLab Self Managed (GLSM) plans.
```

Suggesting:
```
Integrating Semgrep into source code management (SCM) tools
Semgrep App's Team tier supports repositories hosted on GitHub Enterprise (GHE) and GitLab Self Managed (GLSM) plans. When referring to SCM tools, this term means explicitly GHE and GLSM plans.
```

Reply:
Done, thanks :) I just need Ty's review now.